packages/snap-preact-components/src/components/Organisms/Autocomplete/Autocomplete.tsx

```diff
@@ -518,7 +518,7 @@ export const Autocomplete = observer((properties: AutocompleteProps): JSX.Elemen
 											cloneElement(noResultsSlot, { search, pagination, controller })
 										) : (
 											<>
-												<p>No results found for "{search.query.string}".</p>
+												<p>No results found for "{search.originalQuery.string}".</p>
```

Comment:
If I remember right, it is possible that if the query is not corrected `originalQuery` may not exist - we may have to have both in here for those cases:
```
{ search.originalQuery?.string || search.query.string }
```

Reply:
yep